converged_sound_open_fw_modules_development/modules/aca_module/aca_module.cc

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+#if defined(XTENSA_TOOLSCHAIN) || defined(__XCC__)
+#include <xtensa/tie/xt_hifi2.h>
+#endif
+
+#include "aca_module.h"
+
+#include <logger.h>
+
+using namespace intel_adsp;
+
+DECLARE_LOADABLE_MODULE(AcaModule,
+                        AcaModuleFactory)
+
+
+ErrorCode::Type AcaModuleFactory::Create(
+    SystemAgentInterface &system_agent,
+    ModulePlaceholder *module_placeholder,
+    ModuleInitialSettings initial_settings
+    )
+{
+    // count of input pins formats retrieved from the ModuleInitialSettings container
+    const size_t in_pins_format_count = initial_settings.GetItem<IN_PINS_FORMAT>().GetLength();
+    // count of output pins formats retrieved from the ModuleInitialSettings container
+    const size_t out_pins_format_count = initial_settings.GetItem<OUT_PINS_FORMAT>().GetLength();
+
+    LOG_MESSAGE(    LOW, "Create(in_pins_format_count = %d, out_pins_format_count=%d)",
```

Comment:
This should use the trace API.

Reply:
By trace API do you mean SOF trace API?

Reply:
I was initially thinking yes for SOF API, but then we are moving to native Zephyr APIs so logging should be Zephyr (and this saves later effort in aligning).  This could be incrementally done by mapping these ```LOG_``` calls to Zephyr ```printk``` in zephyr/include/sof/trace.h (this means it's buildable for Zephyr and non Zephyr configs)

Reply:
> I was initially thinking yes for SOF API, but then we are moving to native Zephyr APIs so logging should be Zephyr (and this saves later effort in aligning). This could be done by mapping these `LOG_` calls to Zephyr `printk` in zephyr/include/sof/trace.h (this means it's buildable for Zephyr and non Zephyr configs)